src/intl/en/learn-quizzes.json

```
@@ -324,5 +324,41 @@
   "h005-c-label": "Eth1",
   "h005-c-explanation": "Eth1 was the original name given to the execution layer, not the consensus layer.",
   "h005-d-label": "Staking",
-  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain."
+  "h005-d-explanation": "Staking is depositing ETH into a smart contract to help secure the chain.",
+  "k001-prompt": "How is Ethereum scaling?",
+  "k001-a-label": "Layer 2 rollups",
+  "k001-a-explanation": "These help Ethereum scale by bundling transactions, executing them, and then posting the results to Ethereum for validation and securing. Examples or rollups include Arbitrum or Optimism. This is not the only way Ethereum is scaling.",
+  "k001-b-label": "Proto-Danksharding",
+  "k001-b-explanation": "This provides a temporary and inexpensive storage option for saving rollup data to Mainnet, which currently is responsible for approximately 90% of the cost a user encounters on a rollup. This is not the only way Ethereum is scaling.",
+  "k001-c-label": "Danksharding",
+  "k001-c-explanation": "This removes the need for every validator and node on the network from being required to store 100% of the data for all rollups, reducing hardware requirements for node operators. This is not the only way Ethereum is scaling.",
+  "k001-d-label": "All of the above",
+  "k001-d-explanation": "Layer 2 rollups bundle transactions, Proto-Danksharding creates cheap temporary storage for this data, and Danksharding shares the storage burden across all validators—all helping Ethereum scale.",
+  "k002-prompt": "Layer 2 rollups bundle large number of transactions, execute them and then:",
+  "k002-a-label": "Store the data on a private server",
+  "k002-a-explanation": "Results are posted to Mainnet for transparency and public availability, and are not reliant on private servers.",
+  "k002-b-label": "Sends the proof to the user for storage",
+  "k002-b-explanation": "Users are not expected to hold onto the results of their transaction. This information is posted to Mainnet.",
+  "k002-c-label": "Submit the results to Ethereum",
+  "k002-c-explanation": "Layer 2 rollups post the results of their transaction execution to Mainnet, securing it in Ethereum's history",
+  "k002-d-label": "Delete the result to reduce costs",
+  "k002-d-explanation": "Layer 2 rollups post the results of their transaction execution to Mainnet. The cost savings achieved with this approach is by bundling and compressing transaction data, and ultimately storing it in cheap storage that expires once made available to those who need it.",
+  "k003-prompt": "Proto-Danksharding reduces rollup transaction costs by:",
```

Frame it as a question

```suggestion
  "k003-prompt": "How does Proto-Danksharding reduce rollup transaction costs on rollups?",
```